src/AcmSearchbar/AcmSearchbar.tsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react'
+import ReactTags, { Tag } from 'react-tag-autocomplete'
+import CloseIcon from '@patternfly/react-icons/dist/js/icons/times-circle-icon'
+import HelpIcon from '@patternfly/react-icons/dist/js/icons/help-icon'
+import SearchIcon from '@patternfly/react-icons/dist/js/icons/search-icon'
+import './AcmSearchbar.css'
+import { convertStringToTags } from './helper'
+
+export type DropdownSuggestionsProps = Tag & {
+    kind?: 'filter' | 'value' | 'label'
+    disabled?: boolean
+}
+
+type AcmSearchbarProps = {
+    loadingSuggestions: boolean
+    queryString: string
+    suggestions: DropdownSuggestionsProps[]
+    currentQueryCallback: (query: string) => void
+    toggleInfoModal: () => void
+}
+
+export function AcmSearchbar(props: AcmSearchbarProps) {
+    const { loadingSuggestions, suggestions, queryString, currentQueryCallback, toggleInfoModal } = props
+    const [currentQuery, setCurrentQuery] = useState(queryString)
+    const [searchbarTags, setSearchbarTags] = useState(convertStringToTags(currentQuery))
+
+    return (
+        <div className={'searchbar-container'}>
+            <ReactTags
+                placeholder={'Search items'}
```

Comment:
Will need translation.  Do we have a pattern for translating strings?

Reply:
I am not aware of any translation work, @jamestalton  is that planned for?

Reply:
We are setting up translations for console.
Not sure we have a good plan for ui-components translations.